File: Java/Prisma.java
public class Prisma extends Figuras3d{ // es un prisma rectangular 
    protected double largo;
    protected double ancho;
    protected double altura;

    public Prisma(String nombre,double largo, double ancho, double altura){
        super(nombre);
        this.largo=largo;
        this.ancho=ancho;
        this.altura=altura;

    }

    @Override
    public double calcularVolumen(){
        //l*ancho*altura
        return  (largo*ancho*altura); 
        
    }
    
}

File: python/prisma.py
from figuras3d import Figuras3D

class Prisma(Figuras3D):
    def __init__(self, largo, ancho, alto):
        self.largo = largo
        self.ancho = ancho
        self.alto = alto
        
    def calcularVolumen(self):
        volumen = self.largo * self.ancho * self.largo
        return volumen